x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/configuration_step/configuration_step_form.tsx

```diff
@@ -445,6 +456,37 @@ export const ConfigurationStepForm: FC<CreateAnalyticsStepProps> = ({
       >
         <Fragment />
       </EuiFormRow>
+      {Array.isArray(unsupportedRuntimeFields) && unsupportedRuntimeFields.length > 0 && (
+        <>
+          <EuiCallOut size="s" color="warning">
+            <FormattedMessage
+              id="xpack.ml.dataframe.analytics.create.unsupportedRuntimeFieldsCallout"
+              defaultMessage="The runtime {runtimeFieldsCount, plural, one {field} other {fields}} {unsupportedRuntimeFields} {extraCountMsg} are not supported for analysis."
+              values={{
+                runtimeFieldsCount: unsupportedRuntimeFields.length,
+                extraCountMsg:
+                  unsupportedRuntimeFields.length - 5 > 0 ? (
+                    <FormattedMessage
+                      id="xpack.ml.dataframe.analytics.create.extraUnsupportedRuntimeFieldsMsg"
+                      defaultMessage="and {count} more"
+                      values={{
+                        count: unsupportedRuntimeFields.length - 5,
+                      }}
+                    />
+                  ) : (
+                    ''
+                  ),
+                unsupportedRuntimeFields:
+                  unsupportedRuntimeFields.length > 5
+                    ? unsupportedRuntimeFields.slice(0, 5).join(', ')
+                    : unsupportedRuntimeFields.join(', '),
```

Comment:
no need in this check, `slice` doesn't add extra values to an array
```suggestion
                unsupportedRuntimeFields: unsupportedRuntimeFields.slice(0, 5).join(', ')

```

Reply:
Updated here https://github.com/elastic/kibana/pull/90363/commits/7b91ae0a240052507930558a3614119770ddb0b2